tests/Convert-DbaMaskingValue.Tests.ps1

```diff
@@ -0,0 +1,110 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object { $_ -notin ('whatif', 'confirm') }
+        [object[]]$knownParameters = 'Value', 'DataType', 'Nullable'
```

Comment:
Needs `EnableException` added

Reply:
If you want to check the unit test on `Install-DbaFirstResponderKit` we changed the pattern for this test...just have not updated all the other test on it yet.